src/content/advanced/configure-dex-in-your-cluster/index.md

```diff
@@ -92,13 +92,6 @@ metadata:
 data:
   values: |
     isWorkloadCluster: true
-    services:
-      kubernetes:
-        api:
-          caPem: |
-            -----BEGIN CERTIFICATE-----
-            M...=
-            -----END CERTIFICATE-----
```

Comment:
this is now automatically injected in last versions

Reply:
❤️